catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/PolicyRepository.java

```diff
@@ -191,18 +194,32 @@ public void isValid(Policy policy) throws IOException {
       return;
     }
     LOG.debug("Validating rules for {} policy: {}", PolicyType.AccessControl, policy.getName());
+
+    Set<MetadataOperation> operations = new HashSet<>();
     for (Object ruleObject : policy.getRules()) {
       // Cast to access control policy Rule.
       Rule rule = JsonUtils.readValue(JsonUtils.getJsonStructure(ruleObject).toString(), Rule.class);
-      // If the operation is not specified or if all user (subject) and entity (object) attributes are null, the rule
-      // is invalid.
-      if (rule.getOperation() == null
-          || (rule.getEntityTagAttr() == null && rule.getEntityTypeAttr() == null && rule.getUserRoleAttr() == null)) {
+
+      if (rule.getOperation() == null) {
+        throw new IllegalArgumentException(
+            String.format(
+                "Found invalid rule %s within policy %s. Please ensure operation is non-null",
+                rule.getName(), policy.getName()));
+      }
+
+      if (!operations.add(rule.getOperation())) {
```

Comment:
This is the new validation check 

if a duplicate operation is added to a set, it returns false